types/rebass__forms/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for @rebass/forms 4.0
 // Project: https://github.com/rebassjs/rebass#readme
 // Definitions by: zinozzino <https://github.com/zinozzino>
+// Definitions by: trumanshuck <https://github.com/trumanshuck>
```

Comment:
I think the CI error happens because you're not supposed to have two lines in the header with `// Definitions by:`.
```suggestion
//                 trumanshuck <https://github.com/trumanshuck>
```

Reply:
Oh jeeze, thank you so much!